spring-cloud-vault-core/src/main/java/org/springframework/cloud/vault/VaultClient.java

```diff
@@ -170,6 +291,10 @@ private HttpHeaders createHeaders(VaultToken vaultToken) {
 		return headers;
 	}
 
+	private HttpHeaders createHeaders() {
```

Comment:
I'd also missed this (VaultTemplate) as my assumption (sorry, made a lot of those :)) was that everything needed for this change was in the core package.